android-components/components/browser/toolbar/src/main/res/layout/mozac_browser_toolbar_displaytoolbar.xml

```diff
@@ -105,12 +105,24 @@
         android:layout_height="40dp"
         android:layout_marginTop="8dp"
         android:paddingEnd="@dimen/mozac_browser_toolbar_origin_padding_end"
-        app:layout_constraintEnd_toStartOf="@+id/mozac_browser_toolbar_page_actions"
+        app:layout_constraintEnd_toStartOf="@+id/mozac_browser_toolbar_action_separator"
         app:layout_constraintStart_toEndOf="@+id/mozac_browser_toolbar_security_indicator"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_goneMarginStart="8dp"
         app:layout_goneMarginTop="8dp" />
 
+    <View
+        android:id="@+id/mozac_browser_toolbar_action_separator"
+        android:layout_width="@dimen/mozac_browser_toolbar_page_icons_separator_width"
+        android:layout_height="40dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginStart="8dp"
+        android:importantForAccessibility="no"
+        android:scaleType="center"
+        app:layout_constraintStart_toEndOf="@+id/mozac_browser_toolbar_origin_view"
```

Comment:
I have not yet! I am assuming since layout is in xml, we might have to add another similar ImageView before the separator and dynamically make these two hide/visible as needed. What do you think?

Reply:
We could add another view, duplicating it, but also we could create a second xml or rearrange the constraints during init phase. Not sure what's the best one here